### ISSUE TITLE
Update DD4hep Plugin and re-enable extrapolation test

### ISSUE THREAD

**Issue by vvolkl** [HEP-FCC/FCCSW#260]

**Comment by jhrdinka** [comment]
I just freshly checked out the current FCCSW master and the extrapolation is running for me.

**Comment by vvolkl** [comment]
Hi Julia,
that's great! could you please re-enable the test then, as discussed. Although I still get 

```
ExtrapolationTest DEBUG got 0 extrapolation stepsout DEBUG Filling DataTree ..ExtrapolationTest DEBUG start extrapolation ...ExtrapolationTest DEBUG got 0 extrapolation steps
```

no extrapolation steps when I run it,  could you update the test to check for this as well?

**Comment by jhrdinka** [comment]
Ok, I can also check for that.

**Comment by zaborowska** [comment]
I believe this is now addressed by PR #314.
